tensorflow/python/ops/image_ops_impl.py

```diff
@@ -3148,11 +3120,7 @@ def rgb_to_yuv(images):
     ...     [[7.0, 8.0, 9.0],      
     ...       [10.0, 11.0, 12.0]]]
     >>> tf.image.rgb_to_yuv(x)
-    <tf.Tensor: shape=(2, 2, 3), dtype=float32, numpy=
-    array([[[ 1.815    ,  0.5831516, -0.7149856],
-            [ 4.815    ,  0.5831516, -0.7149855]],
-           [[ 7.815    ,  0.5831516, -0.7149856],
-            [10.815001 ,  0.5831518, -0.7149852]]], dtype=float32)>
+    <tf.Tensor: shape=(2, 2, 3), dtype=float32, numpy=...>
```

Comment:
Will the output be random for this on each run?

If not, you can keep values.

Reply:
Yes, but you just said you wanted me to change all of them to this format because of consistency? And also, the last build failed just because I forgot to change the `adjust_hue` to the correct version, the other random outputs haven't given an error.

Reply:
If the float is always the same (`3.14151234`), you should use all of its digits, exactly as it is printed.

If the float is random, you should use `...` instead

Reply:
Sorry for the confusion.

What I said was to not have partial float outputs. So output like `1.95...`. 

For outputs:

* If its going to be random, you can ignore the entire output by replacing it with `...`, like you did for random_brightness.
* For output that's not going to be random and will be constant on every run, you can copy paste it.

All of this is documented here: https://www.tensorflow.org/community/contribute/docs_ref#docstring_considerations

Hope that makes sense. If not, let me know what's confusing and I can document it clearly.

Reply:
Ok, now I feel like there shouldn't be any errors anymore :D